### PR TITLE
Create w10privacy.json

### PR DESCRIPTION
**Pull request by RavenMacDaddy**
Credit goes to [Naderi](https://github.com/naderi/scoop-bucket/tree/f038e1817b6bea4a8753e0fbbce5c2876bfdfc8d) for the original manifest.

I adjusted the description and corrected the hash to the current one, as can be confirmed on the home page for the download [here](https://www.w10privacy.de/deutsch-start/download/).

---

W10Privacy lets you adjust Windows according to your preferences to improve the performance and privacy of the OS.

---

- [ x ] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/w10privacy.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    "shortcuts": [
+        [
+            "W10Privacy.exe",
+            "W10Privacy"
+        ]
+    ],
+    "persist": "W10Privacy.ini",
+    "checkver": {
+        "url": "https://www.winprivacy.de/deutsch-start/download",
+        "regex": "(?sm)current version: v\\.([\\d.]+).*href=\"/app/download/(?<cosi>\\d+)/W10Privacy.zip"
```
Comment (rashil2000) [inline bucket/w10privacy.json:27]: Change `cosi` to a better/understandable variable name

File: bucket/w10privacy.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    },
+    "autoupdate": {
+        "url": "https://www.winprivacy.de/app/download/$matchCosi/W10Privacy.zip"
```
Comment (rashil2000) [inline bucket/w10privacy.json:30]: Same as above for `matchCosi`

**Review by rashil2000 — CHANGES_REQUESTED** [review]

File: bucket/w10privacy.json (hunk 1 above)
Comment (RavenMacDaddy) [reply bucket/w10privacy.json:27]: The stuff after v\\.([\\d.]+). seems kinda redundant, right?

How would you construct the regex when you look at the text on the url?

File: bucket/w10privacy.json (hunk 2 above)
Comment (RavenMacDaddy) [reply bucket/w10privacy.json:30]: To be honest when looking at the resulting download URL I'm not sure how to change it to something else, it's seemingly just a row of numbers;

https://www.w10privacy.de/app/download/12302828636/W10Privacy.zip?t=1633760316

File: bucket/w10privacy.json (hunk 1 above)
Comment (rashil2000) [reply bucket/w10privacy.json:27]: Use `randnum`

Comment (RavenMacDaddy) [reply bucket/w10privacy.json:27]: And then for the text below: $matchRandnum?

Comment (rashil2000) [reply bucket/w10privacy.json:27]: Yes

Comment (RavenMacDaddy) [reply bucket/w10privacy.json]: Adjusted, does it look good?

Comment (rashil2000) [reply bucket/w10privacy.json]: Yes

**Comment by rashil2000** [comment]
Please follow the contributing guide on how to select PR titles from next time.

**Comment by RavenMacDaddy** [comment]
> Please follow the contributing guide on how to select PR titles from next time.

I had some difficulty interpreting the information on that page, but I'll follow the structure of the other commits for reference in the future.

**Pull request merged**